### ISSUE TITLE
can not reach your results of val error 1.83 on KITTI15

### ISSUE THREAD

**Issue by shanyuhu** [JiaRenChang/PSMNet#73]
Hi @JiaRenChang 
I pretrained your stackhourglass network on sceneflow dataset for 10 epoch, and then finetune it on KITTI15 with your provided train/val split for another 300 epoches. However, I can only get 2.8% error rate which is far from your result of 1.83%. All settings are the default ones provided in your code.
Did I miss anything? Looking forward to your reply.

**Comment by JiaRenChang** [comment]
@shanyuhu 
Did you check the validation set are those ones? [1,3,6,20,26,35,38,41,43,44,49,60,67,70,81,84,89,97,109,119,122,123,129,130,132,134,141,144,152,158,159,165,171,174,179,182, 184, 186,187,196]

**Comment by shanyuhu** [comment]
Yes, I use the same validation set as yours. 
One difference is that I use larger batch sizes instead of 12 during training. Do you think this will be the reason?

**Comment by JiaRenChang** [comment]
@shanyuhu 
We tried batch size = 32,16,12,8,4 in our experiment.
We found that the best performance on training scene flow is using batch size = 8 or 12.

It may related to that pytorch updated their upsample function.
I am not sure.

**Comment by shanyuhu** [comment]
@JiaRenChang thanks for your reply. Btw, whether the provided kitti15 model is the one you used for the final kitti15 leaderboard submission?

**Comment by JiaRenChang** [comment]
@shanyuhu 
That pretrained model is our submission model.

**Comment by michael-peng** [comment]
@JiaRenChang Is the feature extractor in the sceneflow model pre-trained using other dataset, like imagenet?

**Comment by JiaRenChang** [comment]
@michael-peng 
No, it was trained from scratch.

**Comment by givasile** [comment]
> @shanyuhu
> Did you check the validation set are those ones? [1,3,6,20,26,35,38,41,43,44,49,60,67,70,81,84,89,97,109,119,122,123,129,130,132,134,141,144,152,158,159,165,171,174,179,182, 184, 186,187,196]

you refer to indexes in [0,199] or [1, 220]? I suppose the first, am I correct?

**Comment by JiaRenChang** [comment]
@givasile 
Yes, [0,199].

**Comment by gongwei1993** [comment]
Could you share your finetune PSMNet with me? As my computer's memory is not enough?

**Comment by Chenlei6** [comment]
Can you tell me how to divide the training set and validate set on KITTI2012, preferably the picture number, thank you!

**Comment by JiaRenChang** [comment]
@Chenlei6 
kitti 2012 val set
[3,6,20,26,38,41,43,44,49,60,67,70,81,84,89,97,109,119,122,123,129,130,132,134,141,144,152,158,165,171,174,179,184,186]

**Comment by Chenlei6** [comment]
@JiaRenChang Thank you！

**Comment by Chenlei6** [comment]
I‘m sorry to disturb you! Can you share the training method for getting the best model on the KITTI2015 dataset? In the paper , you said that you have trained 1000 epochs. Is there any other training strategy to share?
